### PR TITLE
Feat/sepolia testnet

### PR DESCRIPTION
**Pull request by foodaka**
## General Changes

- Adds support for sepolia testnet

## Developer Notes

Add any notes here that may be helpful for reviewers.

---

## Author Checklist

Please ensure you, the author, have gone through this checklist to ensure there is an efficient workflow for the reviewers.

- [x]  The base branch is set to `main`
- [x]  The title is using [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting
- [ ]  The Github issue has been linked to the PR in the Development section
- [x]  The General Changes section has been filled out
- [ ]  Developer Notes have been added (optional)

**If the PR is ready for review:**

- [x]  The PR is in `Open` state and not in `Draft` mode
- [x]  The `Ready for Dev Review` label has been added

## Reviewer Checklist

Please ensure you, as the reviewer(s), have gone through this checklist to ensure that the code changes are ready to ship safely and to help mitigate any downstream issues that may occur.

- [ ]  End-to-end tests are passing without any errors
- [ ]  Code style generally follows existing patterns
- [ ]  Code changes do not significantly increase the application bundle size
- [ ]  If there are new 3rd-party packages, they do not introduce potential security threats
- [ ]  If there are new environment variables being added, they have been added to the `.env.example` file as well as the pertinant `.github/actions/*` files
- [ ]  There are no CI changes, or they have been approved by the DevOps and Engineering team(s)
- [ ]  Code changes have been quality checked in the ephemeral URL
- [ ]  QA verification has been completed
- [ ]  There are two or more approvals from the core team
- [ ]  Squash and merge has been checked

### CODE REVIEW

File: src/ui-config/marketsConfig.tsx [hunk 1]
```diff
@@ -237,6 +238,23 @@ export const marketsData: {
       UI_INCENTIVE_DATA_PROVIDER: markets.AaveV3Goerli.UI_INCENTIVE_DATA_PROVIDER,
     },
   },
+  [CustomMarket.proto_sepolia_v3]: {
```
Comment (grothem) [inline src/ui-config/marketsConfig.tsx:241]: Maybe we should add this in the list before goerli show it shows up first and as the default v3 testnet?

Comment (foodaka) [reply src/ui-config/marketsConfig.tsx:224]: good call- updated in `8c664765204adcd9339b259ae89eb1932da6db6b`

File: src/ui-config/networksConfig.ts [hunk 2]
```diff
@@ -69,6 +69,25 @@ export const networkConfigs: Record<string, BaseNetworkConfig> = {
...
+      'https://rpc.sepolia.online',
+      'https://www.sepoliarpc.space',
+    ],
+    // publicJsonRPCWSUrl: 'wss://eth-goerli.public.blastapi.io',
+    // protocolDataUrl: '',
+    baseUniswapAdapter: '0x0',
+    baseAssetSymbol: 'ETH',
+    wrappedBaseAssetSymbol: 'WETH',
+    baseAssetDecimals: 18,
+    explorerLink: 'https://goerli.etherscan.io',
```
Comment (defispartan) [inline src/ui-config/networksConfig.ts:86]: Should be https://sepolia.etherscan.io/

Comment (foodaka) [reply src/ui-config/networksConfig.ts]: `46a5acb60ae6688ce553a713c84135ba259ccd67`

**Review by grothem — APPROVED** [review]

**Review by 0x4Graham — APPROVED** [review]

**Pull request merged**